Formula/nushell.rb

```diff
@@ -0,0 +1,28 @@
+class Nushell < Formula
+  desc "Modern shell for the GitHub era"
+  homepage "https://www.nushell.sh"
+  url "https://github.com/nushell/nushell/archive/0.2.0.tar.gz"
+  sha256 "5bce8cdb33a6580ff15214322bc66945c0b4d93375056865ad30e0415fece3de"
+  head "https://github.com/nushell/nushell.git"
+
+  depends_on "openssl"
+
+  # Nu requires features from Rust 1.39 to build, so we can't use Homebrew's
+  # Rust; picking a known-good Rust nightly release to use instead.
+  resource "rust-nightly" do
```

Comment:
I wonder if we need a nightly formula, as miniserve formula also needs this.

Reply:
Good question. Maybe a rust-nightly formula that we bump only every month or so? That way we'd have the option to build from source, too, so we're not using a binary like here.

Reply:
Indeed, I know racer also needs a Rust nightly build. 
https://github.com/racer-rust/racer/tree/v2.1.2#installation